packages/shared/src/components/squads/SquadJoinButton.tsx

```diff
@@ -92,6 +94,7 @@ export const SquadJoinButton = ({
       onError: () => {
         displayToast(labels.error.generic);
       },
+      onSuccess: onSuccess && (() => onSuccess()),
```

Comment:
This should work like this, right?
```suggestion
      onSuccess,
```